curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-072.md

```diff
@@ -1,22 +1,27 @@
 ---
-id: 5f459cf202c2a3472fae6a9f
+id: 5f459fd48bdc98491ca6d1a3
 title: Part 72
 challengeType: 0
 dashedName: part-72
 ---
 
 # --description--
 
-Notice how the thickness of the line looks bigger? The default value of a property named `border-width` is `1px` for all edges of `hr` elements. By changing the background to the same color as the background, the total height of the line is `5px` (`3px` plus the top and bottom border width of `1px`).
-
-Change the `height` property of the `hr` to be `2px`, so the total height of it becomes `4px`.
+Go ahead and add another `hr` element between the `main` element and the `footer` element.
 
 # --hints--
 
-Test 1
+You should add a second `hr` element.
 
 ```js
+assert($('hr').length === 2);
+```
 
+Your new `hr` element should be between the `main` element and the `footer` element.
+
+```js
+assert($('hr')[1].previousElementSibling.tagName === 'MAIN');
+assert($('hr')[1].nextElementSibling.tagName === 'FOOTER');
```

Comment:
`<hr></hr>` passes

Reply:
So at this point, we've already taught the camper that `hr` elements are self-closing and we've asserted they use the correct syntax.

I think, rather than asserting it again, it's better to allow code to pass if it renders correctly (and `<hr></hr>` does). Thoughts?

Reply:
That makes sense.